### ISSUE TITLE
Graylog web interface stops interact

### ISSUE THREAD

**Issue by andbus** [billmurrin/graylog-plugin-quickvaluesplus-widget#25]
When plugin is installed the Graylog web interface stops properly responding. Clicking on any button works only once. To restore communication with the web interface I need to refresh browser page (Ctrl-R or F5). Then, click works only once and so on.

Graylog: 2.3.0-7
Plugin: 3.0.0
JDK: 1.8.0

**Comment by billmurrin** [comment]
Hi @andbus, can you please provide what OS and browser version you are using?

Have you tried clearing the cache on your browser and/or force reloading the page (Ctrl + F5) to reload the cached assets?

If you open up the page inside of the developer tools (typically F12), do you see any error messages occurring when you Clicking on any button?

By button, do you mean a button inside of the widget or the entire Graylog web interface? 

I'm sorry for the inconvenience but appreciate you taking the time to report the issue.

**Comment by andbus** [comment]
@billmurrin 

I'm using Win7 and latest version of Opera and Firefox on the client side. And yes, I've tried to force clearing the cache and page. Regarding cache/page clearing the situation is as described (works for one click only). However, if I press Search button for another query, I can observe that Search result (number of messages) is changing but all other elements of the page stay intact (I mean Histogram, Messages etc).

I can't reproduce the error now, because currently I had to remove plugin from test environment.

Regarding button, I mean general interactive elements of Graylog web interface. I don't know if all buttons because I've checked it superficially.

**Comment by andbus** [comment]
[GraylogWithQVP-OperaConsole-1503757010828.log.txt](https://github.com/billmurrin/graylog-plugin-quickvaluesplus-widget/files/1253742/GraylogWithQVP-OperaConsole-1503757010828.log.txt)
@billmurrin 

Fortunately,  I was able to test the plugin. Attached are the logs of the browser console.

**Comment by billmurrin** [comment]
@andbus - I will attempt to resolve the error you are receiving. Please note that when I installed the latest versions of Opera and Firefox I couldn't recreate the error. With that said, I have a pretty good idea where it might take place. I'll give you an update soon. Thanks.

**Comment by billmurrin** [comment]
@andbus - I packaged a version with a few additional console.log commands, can you run this version and recreate the event and then and provide the logs.

https://www.dropbox.com/s/om256jky1bwnt2v/graylog-plugin-quickvaluesplus-widget-3.0.0.jar?dl=0

**Comment by billmurrin** [comment]
@andbus - Can you please go into your System configuration and update/save the Configuration values for the widget and then try again? Thanks.

**Comment by billmurrin** [comment]
Hi @andbus, please update to version 3.0.1 of the plugin. Thanks for the report. The issue was related to a plugin migration, but I saw another issue and fixed that as well.
